lib/noop.js

```diff
@@ -0,0 +1 @@
+module.exports = require('lodash/utility/noop')
```

Comment:
It does not I checked

On Mon, May 4, 2015, 8:05 AM Nolan Lawson <notifications@github.com> wrote:

> In lib/noop.js
> <https://github.com/tgriesser/knex/pull/810#discussion_r29579429>:
>
> > @@ -0,0 +1 @@
> > +module.exports = require('lodash/utility/noop')
>
> But what if lodash has performance improvements?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgriesser/knex/pull/810/files#r29579429>.
>